augur/export_v2.py

```diff
@@ -819,17 +819,17 @@ def node_data_prop_is_normal_trait(name):
     return True
 
 
-def register_arguments_v2(subparsers):
-    v2 = subparsers.add_parser("v2", help="Export version 2 JSON schema")
+def register_parser(parent_subparsers):
+    parser = parent_subparsers.add_parser("v2", help="Export version 2 JSON schema")
```

Comment:
_(minor, non-blocking)_

```suggestion
    parser = parent_subparsers.add_parser("v2", help=__doc__)
```

and update the module docstring to be `Export version 2 JSON schema`. Similarly for `export_v1.py`.

Reply:
Done in 6df2066